code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -1,3 +1,11 @@
+#define END_FERMI_REACTION \
+    do { \
+        if(beaker.reagents.fermiIsReacting) { \
+            beaker.reagents.fermiEnd() } \
+    } while(0)
```

Comment:
Don't hide functionality like this. Learned that lesson the hard way.

Reply:
it was a proc before, then @ExcessiveUseOfCobblestone told me to change it to a define. should i make it a proc again?

Reply:
no reason to have it as a proc, it's just shorthand because you don't want to c/v it everywhere.